### PR TITLE
cinnamon: make it true in rules and refine assets

### PR DESCRIPTION
**Pull request by Muqtxdir**
- use true for `cinnamon` in `rules` for `yaru-theme-gtk.install`
- improve/refine assets under `cinnamon/`:
  - [x] use `16x16` for checkboxes and `26x26` for close.
  - [x] improve `checkbox-*`, `switch-*`, `calendar-*`, `close-*`

### CODE REVIEW

**Review by Jupi007 — COMMENTED** [review]
Hi @Muqtxdir 👋
Thanks for working on this.

I added some comments :)

File: debian/yaru-theme-gtk.install [hunk 1]
```diff
@@ -4,10 +4,12 @@ usr/share/themes/Yaru/gtk-3.0/
...
 usr/share/themes/Yaru/metacity-1
 usr/share/themes/Yaru/xfwm4
+usr/share/themes/Yaru/cinnamon
 usr/share/themes/Yaru/index.theme
 usr/share/themes/Yaru-dark/gtk-2.0/
 usr/share/themes/Yaru-dark/gtk-3.0/
 usr/share/themes/Yaru-dark/gtk-4.0/
 usr/share/themes/Yaru-dark/metacity-1
 usr/share/themes/Yaru-dark/xfwm4
+usr/share/themes/Yaru-dark/cinnamon
```
[the remark this answers is not in the document]
Comment (ItzSwirlz) [reply debian/yaru-theme-gtk.install:14]: If we did for xfce we should here

**Comment by ItzSwirlz** [comment]
This would be great. This weekend I'll review the theme and nitpick issues

File: debian/yaru-theme-gtk.install (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jupi007) [reply debian/yaru-theme-gtk.install:14]: Well that very different in fact.
Yaru `xfce` component, are just some assets to have correct headerbars.

Here with cinnamon we have a brand new desktop theme.
IMO we should add a new package.

**Comment by ItzSwirlz** [comment]
Am I the only one who sees the Bluetooth icon as blurry?image

File: debian/yaru-theme-gtk.install (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jupi007) [reply debian/yaru-theme-gtk.install:14]: @Muqtxdir Everything looks good for me, except this ;)

**Review by Jupi007 — CHANGES_REQUESTED** [review]

**Comment by Muqtxdir** [comment]
I didn't do this because it may conflict and confusion with packages from Ubuntu Cinnamon project, since there already exists a package named similarly as `yaru-theme-cinnamon` series. (_I think there was an issue like this_)

**Comment by Jupi007** [comment]
Ah, this is annoying...
I don't think we should wrap such a big thing like the cinnamon theme in `yaru-theme-gtk`.

It is possible to make a kind of migration with the Ubuntu Cinnamon's package?

**Comment by Muqtxdir** [comment]
> It is possible to make a kind of migration with the Ubuntu Cinnamon's package?

It's possible to do this but Im not aware with full debian package replacement. @ItzSwirlz could you please help us out on this one? it looks like you maintain packages for ubuntu-cinnamon

**Comment by ItzSwirlz** [comment]
> > It is possible to make a kind of migration with the Ubuntu Cinnamon's package?
> 
> 
> 
> It's possible to do this but Im not aware with full debian package replacement. @ItzSwirlz could you please help us out on this one? it looks like you maintain packages for ubuntu-cinnamon

I didn't know dh migratiosn exist. I moved Yaru-theme-cinnamon-* as part of the ubuntucinnamon-environment package. It's PPA though. I can rename my cinnamon variant to Yaru-cinnamon-theme-* and a migration or something will be setup

**Comment by Jupi007** [comment]
Thanks for your work on this @Muqtxdir :+1:

**Pull request merged**